int32 freq
int32 C
---
bool ACK
